assets/queries/k8s/liveness_probe_not_defined/query.rego

```diff
@@ -0,0 +1,18 @@
+package Cx
+
+CxPolicy [ result ] {
+   document := input.document[i]
+   spec := document.spec
+   metadata := document.metadata
+   containers := spec.containers
+   exists_liveness_probe = object.get(containers[index], "livenessProbe", "undefined") == "undefined"
```

Comment:
When the resource is a "Job" or "CronJob", they aren't required to have a liveness probe, since they are meant to schedule their own tasks and check if they are valid. https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#cron-job-limitations